src/devices/Ft4222/FtFunction.cs

```diff
@@ -12,15 +12,133 @@ namespace Iot.Device.Ft4222
     /// </summary>
     internal class FtFunction
     {
-        #region ftd2xx.dll
+        #region ftd2xx wrapper
 
         /// <summary>
         /// Create Device Information List
         /// </summary>
         /// <param name="numdevs">number of devices</param>
         /// <returns>The status</returns>
-        [DllImport("ftd2xx")]
-        public static extern FtStatus FT_CreateDeviceInfoList(out uint numdevs);
+        public static FtStatus FT_CreateDeviceInfoList(out uint numdevs)
+        {
+            if (Environment.OSVersion.Platform == PlatformID.Unix)
```

Comment:
optional: I'm wondering if you should check for Windows first since Unix implementation is more likely to work on any non-Windows platforms.

Reply:
The thing is that this implementation should work for MacOS as well. As on Mac OS as on Windows, there there are 2 different libs.